docs/REST API/rest-api.md

```diff
@@ -9048,6 +9048,116 @@ Content-Type: application/octet-stream
 
 `Status: 200`
 
+### Modify package
+
+_Available in Fleet Premium._
+
+Update a package to install on macOS, Windows, or Linux (Ubuntu) hosts.
+
+
+`PATCH /api/v1/fleet/software/titles/:title_id`
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                      |
+| ----            | ------- | ---- | --------------------------------------------     |
+| software        | file    | form | Optional. Installer package file. Supported packages are PKG, MSI, EXE, and DEB.   |
```

Comment:
I think we only specify when things are required (otherwise assumed to be optional).
```suggestion
| software        | file    | form | Installer package file. Supported packages are PKG, MSI, EXE, and DEB.   |
```